src/client/runner/automation/utils/get-element.js

```diff
@@ -0,0 +1,73 @@
+import hammerhead from '../../deps/hammerhead';
+import testCafeCore from '../../deps/testcafe-core';
+import Cursor from '../cursor';
```

Comment:
I have a note that there is such thing as a `static class`. You have a singleton, which is in JS is basically an object.